SPECS/tdnf/tdnf.spec

```diff
@@ -167,9 +166,23 @@ if [[ ! -f %{_tdnf_history_db_dir}/history.db ]]; then
     %{_libdir}/tdnf/tdnf-history-util init
 fi
 
+%preun
+if  [[ $(readlink $(rpm --eval %{_bindir})/yum) == $(rpm --eval %{_bindir})/tdnf ]]; then
+  rm $(rpm --eval %{_bindir})/yum
+fi
+
```

Comment:
This is *only* needed for the uninstall (not upgrade) action, right? In that case I think this also needs to check `$1 == 0` to make sure we're executing uninstall instead of upgrade.

Also if the symlink is changed to a relative symlink, then the right half of the symlink target check would just be `== tdnf`.

Reply:
Yes, correct, this `rm` action is only needed on uninstall. Will add the check for uninstall only.
Also will add the relative symlink change since I agree it will be good to avoid absolute symlinks for files in the same directory